### PR TITLE
Modified the UI of Upload Form

### PR DESCRIPTION
**Pull request by Sristi27**
I have worked on the UI of the Upload Form.Worked on Image Preview before addition and also Form Reset  after successful submission.

Do check it out the work and let me know incase of any issues!

Thanks for the opportunity!

### CODE REVIEW

File: src/ImageUpload.js [hunk 1]
```diff
@@ -48,23 +94,78 @@ function ImageUpload({ username }) {
...
 
   return (
-    <div className="imageupload">
-      <progress className="imageupload__progress" value={progress} max="100" />
-      <Input
+
+    //Upload Form UI changed
+    <Card className="imageupload">
+      <CardContent>
+        <Typography variant="h5" style={{textAlign:'center',marginBottom:'10px'}}>Add Post</Typography>
```
Comment (abhishek2x) [inline src/ImageUpload.js:109]: Mention a className here and add styling in useStyle const. It's a good practise.

Comment (Sristi27) [reply src/ImageUpload.js:109]: Okay making the changes!

**Comment by abhishek2x** [comment]
Hey @Sristi27, just go through the added comments and make the necessary changes. Rest LGTM.

**Comment by Sristi27** [comment]
Hello,Mentors please look into this.Its not merged yet.

**Review by avinashkranjan — APPROVED** [review]

**Pull request merged**